tests/PHPStan/Rules/Properties/UninitializedPropertyRuleTest.php

```diff
@@ -15,8 +18,29 @@ class UninitializedPropertyRuleTest extends RuleTestCase
 
 	protected function getRule(): Rule
 	{
+		$containerMock = $this->createMock(Container::class);
+		$containerMock->method('getServicesByTag')
```

Comment:
Instead of mocking, please create a separate test case similar to this one, but also create a new `.neon` file next to the test, and configure the extension there.

And then reference the new `.neon` file in `public static function getAdditionalConfigFiles()` in the test case 👍